src/main/cpp/FfmpegSdlAvPlayback.cpp

```diff
@@ -110,7 +110,7 @@ FfmpegSdlAvPlayback::FfmpegSdlAvPlayback(int startup_volume)
               SDL_MIX_MAXVOLUME);
 }
 
-FfmpegSdlAvPlayback::~FfmpegSdlAvPlayback() { av_free(window_title); }
```

Comment:
I think the right solution would be
// Note, that the p_window_title can be null if no video was opened
if (p_window_title != nullptr) {
    av_free(window_title);
}

+ In the constructor add
p_window_title(nullptr)